python/GafferSceneUI/PathFilterUI.py

```diff
@@ -114,6 +118,56 @@
 
 )
 
+##########################################################################
+# VectorDataPlugValueWidget customisation
+###########################################################################
+
+class _PathsPlugValueWidget( GafferUI.VectorDataPlugValueWidget ) :
+
+	def __init__( self, plug, **kw ) :
+
+		GafferUI.VectorDataPlugValueWidget.__init__( self, plug, **kw )
+
+		self.vectorDataWidget().dataMenuSignal().connect( Gaffer.WeakMethod( self.__dataMenu ), scoped = False )
+
+	def __dataMenu( self, vectorDataWidget, menuDefinition ) :
+
+		selectedIndices = vectorDataWidget.selectedIndices()
+
+		if len( selectedIndices ) > 0 :
+			menuDefinition.append( "/divider", { "divider" : True } )
```

Comment:
Because this has the same name as the divider created by VectorDataWidget, it's removing the divider above the "Delete Rows" menu item. We need to give them unique names so we get both dividers.

Reply:
Fixed in 3e96fded732a4d166ed6733e5b197d86d51ed308